### PR TITLE
Let's Encrypt rate limits

### PR DESCRIPTION
**Pull request by kylemclaren**
Folks sometimes run into issues with Let's Encrypt [rate limits](https://letsencrypt.org/docs/rate-limits/) when adding custom domains to their Fly apps.

_Possibly_ related community posts:

https://community.fly.io/t/certificate-stuck-at-awaiting-certificates/6342/30
https://community.fly.io/t/delayed-awaiting-certificates/14628/2
https://community.fly.io/t/issuing-a-certificate-stuck-awaiting/8009
https://community.fly.io/t/generating-an-ssl-certificate-verified-but-rsa-or-ecdsa-never-generated/2303

When this happens, customer impact is high, but there is not much visibility on their end. This doc update provides some context on these limits and gives basic troubleshooting steps, as well as breaking the bad news that all you can really do after hitting these limits is _wait_.

The PR also moves to using Fly.io instead of Fly in certain places, per internal style guide.

### CODE REVIEW

File: app-guides/custom-domains-with-fly.html.md [hunk 1]
```diff
@@ -401,11 +400,27 @@ The GraphQL mutation returns the app name and the hostname and certificate id th
...
+
+The following primary [rate limits](https://letsencrypt.org/docs/rate-limits/) from Let's Encrypt apply:
+
+* **Certificates per Registered Domain**: 50 per week
+* **Duplicate Certificate limit**: 5 per week
+
+If you encounter issues when creating or validating a certificate for a custom domain on Fly.io, you can use the following methods to troubleshoot:
+
+* **Use Let's Debug**: Visit [Let's Debug](https://letsdebug.net/) to diagnose the issue. Let's Debug is a tool that tests for issues with your domain's configuration or issues related to Let's Encrypt's service. Enter your domain name and run the test. The tool will provide a detailed report of any problems it finds and suggest possible solutions.
+* **Wait and Retry**: If you've hit a rate limit, you'll need to wait until the rate limit window passes before you can successfully create or validate a certificate again. We don’t have a way to reset it. The length of the wait depends on the specific rate limit you've hit.
```
Comment (andie787) [inline app-guides/custom-domains-with-fly.html.md:419]:
```suggestion
* **Wait and Retry**: If you've hit a rate limit, you'll need to wait until the rate limit window passes before you can successfully create or validate a certificate again. We don’t have a way to reset it. 
```

Suggest removing that bit about the length, or rewording to say it depends on when the limit was hit and when they started creating or renewing certs?

**Review by andie787 — APPROVED** [review]
This is great info! 
Left one suggestion.

**Comment by kylemclaren** [comment]
Thanks @andie787! Agree with that...

Might add a little more info about [Failed Validation Limit](https://letsencrypt.org/docs/failed-validation-limit/), too.

**Comment by kylemclaren** [comment]
https://fly.io/docs/app-guides/custom-domains-with-fly/#certificate-creation-or-validation-seems-to-hang-stall-or-fail

**Comment by redjonzaci** [comment]
Looking great!

**Comment by redjonzaci** [comment]
Caught a typo:
Note that 
***certficate***
renewals don’t count against your Certificates per Registered Domain limit.

**Pull request merged**